jwst/datamodels/dqflags.py

```diff
@@ -105,3 +106,32 @@ def interpret_bit_flags(bit_flags, flip_bits=None):
         bit_flags_dm = multiple_replace(bit_flags, dm_flags)
 
     return ap_interpret_bit_flags(bit_flags_dm, flip_bits=flip_bits)
+
+
+def bit_flags_to_mnemonics(bit_flags):
+    """Interpret value as bit flags and return the mnemonics
+
+    Parameters
+    ----------
+    bit_flags : int-like
+        The value to interpret as bit flags
+
+    Returns
+    -------
+    mnemonics : [str[,...]]
```

Comment:
Would it be helpful to return these as a `set` instead of a `list`?  Not that the list will ever be long enough to gain any efficiency there (membership test may even be slower for such a short list), but the set intersection/union/etc methods might be handy.